google-cloud-spanner/src/main/java/com/google/cloud/spanner/encryption/EncryptionConfig.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spanner.encryption;
+
+import com.google.api.core.InternalApi;
+
+/**
+ * Marker interface for encryption configurations that can be applied on databases, backups and
+ * restores.
+ */
+@InternalApi
+public interface EncryptionConfig {}
```

Comment:
I don't think we need this interface. It seems that it is only referenced by the other interfaces that extend this, but this specific marker interface is never used.

Reply:
Removed.